sklearn/inspection/_plot/tests/test_plot_partial_dependence.py

```diff
@@ -34,13 +35,19 @@ def clf_diabetes(diabetes):
     return clf
 
 
+def test_plot_partial_dependence_deprecation(pyplot, clf_diabetes, diabetes):
```

Comment:
Could you isolate all the test of the `plot_partial_dependence` in a py file. Following what I did for the other, the file would be name `test_partial_dependence_display.py`

Then you can let the `test_plot_partial_dependence.py` untouched, only adding a filter for the warnings on the top of the file:

```python
pytestmark = pytest.mark.filterwarnings(
    # TODO: Remove in 1.2 (as well as all the tests below)
    "ignore:Function plot_partial_dependence is deprecated",
)
```